app/graphql/recommendation.py

```diff
@@ -12,7 +12,7 @@
     model=RecommendationModel,
     # Currently, the `strawberry.federation.type` cannot be combined with `strawberry.experimental.pydantic.type`,
     # so set the key as a directive.
-    directives=[Key("item { itemId }", UNSET)],
+    directives=[Key(fields="item { itemId }", resolvable=UNSET)],
```

Comment:
@mmiermans yes, this thing failed on `docker-compose up` for me, I wonder how CI missed it

Reply:
I thought it was caused by the FastAPI upgrade, but that wouldn't explain why it happened to you. :man_shrugging: